src/Workspaces/Core/Portable/Microsoft.CodeAnalysis.Workspaces.csproj

```diff
@@ -292,7 +292,7 @@
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.CSharp.Workspaces.UnitTests" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.VisualBasic.Workspaces.UnitTests" />
     <InternalsVisibleTo Include="Microsoft.VisualStudio.LanguageServices.CSharp.UnitTests" />
-    <InternalsVisibleTo Include="Roslyn.VisualStudio.Closed.UnitTests" />
+    <InternalsVisibleTo Include="Roslyn.VisualStudio.Closed.UnitTests" WorkItem="https://github.com/dotnet/roslyn/issues/35081" />
```

Comment:
This assembly is just dead I'm guessing.

Reply:
Left for a follow-up where we can make sure this is correct. #35081